commands/provider/service_deployer.go

```diff
@@ -56,7 +56,7 @@ func (p *ServiceProvider) ServiceDeploy(path string, env map[string]string, stat
 	}()
 	go readDeployReply(stream, deployment, statuses)
 
-	if govalidator.IsURL(path) {
+	if _, err := url.Parse(path); err != nil {
```

Comment:
I think, `url.Parse()` is not enough, it doesn't require a url schema and won't return an error.

https://play.golang.org/p/g_5OQhzyWv7